packages/chains/deployments/ganache.json

```diff
@@ -3,7 +3,7 @@
   "chainId": "1337",
   "contracts": {
     "AddressesProvider": {
-      "address": "0x5B3639BaDD3A08da48cBfb8F8451ff0035d9a4c8",
```

Comment:
This whole file can be reverted. It will always change for you as you are using a different private key.